src/mainnet/compounders/balancer/AuraBalCompounder.sol

```diff
@@ -1,295 +1,295 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.17;
 
-// ███████╗░█████╗░██████╗░████████╗██████╗░███████╗░██████╗░██████╗
```

Comment:
I presume all of these changes are because of the outstanding unit testing?

Reply:
yep